### ISSUE TITLE
Extends the templating fields for dependencyDashboardFooter

### ISSUE THREAD

**Issue by ademar59** [renovatebot/renovate#13345]
### What would you like Renovate to be able to do?

Hi,

I want to customize the footer on dependency dashboard with the project url.

But templating is not available for this parameter : https://docs.renovatebot.com/templates/

### If you have any ideas on how this should be implemented, please tell us here.

I could be nice to have all the dependencyDashboard parameter with templating : title, header, footer

A more complicated idea is to have the same config as `prBodyTemplate` with {{header}}}{{{table}}}{{{notes}}}{{{changelogs}}}{{{configDescription}}}{{{controls}}}{{{footer}}}

Something like `dependencyDashboardBodyTemplate`

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
Is "project URL" already a field supported in templates? If not then what is it?

**Comment by ademar59** [comment]
It is the project path you can set to RENOVATE_EXTRA_FLAGS to analyse a project

ex: renovate-bot/renovate-runner for https://gitlab.com/renovate-bot/renovate-runner

**Comment by rarkins** [comment]
I still don't understand what you mean

**Comment by ademar59** [comment]
The gitlab variables name is CI_PROJECT_PATH

https://docs.gitlab.com/ee/ci/variables/predefined_variables.html

In the documentation of renovate in gitlab, you call it `project name` https://gitlab.com/renovate-bot/renovate-runner
```
If you wish to manually specify which projects that your bot runs again, then add this variable with a space-delimited set of project names: RENOVATE_EXTRA_FLAGS: group1/repo5 user3/repo1.
```

**Comment by rarkins** [comment]
Those are just a list of repositories. Renovate runs on one at a time. The project URL env is not part of renovate so is not something we would pass through.

**Comment by ademar59** [comment]
In the log info, this is logged as repository

 INFO: Repository started (repository=mobivia/driverservices/cloud-platform/digipaas-sandbox/digipaas-sandbox-ops)
       "renovateVersion": "31.14.1"
 INFO: Dependency extraction complete (repository=mobivia/driverservices/cloud-platform/digipaas-sandbox/digipaas-sandbox-ops)
       "baseBranch": "master",
       "stats": {
         "managers": {
           "gitlabci-include": {"fileCount": 1, "depCount": 1},
           "helmv3": {"fileCount": 1, "depCount": 2}
         },
         "total": {"fileCount": 2, "depCount": 3}
       }
 INFO: Branch updated (repository=mobivia/driverservices/cloud-platform/digipaas-sandbox/digipaas-sandbox-ops, branch=renovate/minor)
       "commitSha": "e5398d9"
 INFO: Repository finished (repository=mobivia/driverservices/cloud-platform/digipaas-sandbox/digipaas-sandbox-ops)
       "durationMs": 15377

**Comment by rarkins** [comment]
Yes, and repository is exposed to templates so you could use that

**Comment by ademar59** [comment]
I don't understand what means 👎 

What's the solution?
